### PR TITLE
Update Carousels

### PR DESCRIPTION
**Pull request by kenyonbowers**
Yeah.

### CODE REVIEW

File: src/devotionals/components/AuthorPreviewColumn.vue [hunk 1]
```diff
@@ -19,14 +19,14 @@ import PocketBaseClient from '../../api/PocketBaseClient'
...
+import { getBucketUrl } from '../../api/BucketStorageService'
 
 export interface AuthorPreviewColumnProps {
   devoObject?: Devotional
 }
 const props = defineProps<AuthorPreviewColumnProps>()
 
-//@ts-expect-error
-var imgURL:any = PocketBaseClient.records.getFileUrl(props.devoObject, props.devoObject?.coverImage, {});
+var imgURL:any = getBucketUrl(props.devoObject, props.devoObject?.coverImage, {});
```
Comment (tt9) [inline src/devotionals/components/AuthorPreviewColumn.vue:29]: Change var to const

File: src/home/pages/Home.vue [hunk 2]
```diff
@@ -57,18 +94,25 @@
...
-
     <!-- Latest Devotionals -->
     <div class="relative p-2 md:p-8 mb-16">
-      <h2 class="font-bold font-title mb-8 md:mb-16 text-4xl">Recommended Sermons</h2>
-      <ContentCarousel :slides="sermonSlides"></ContentCarousel>
+      <h2 class="font-bold font-title mb-8 md:mb-16 text-4xl">
+        Recommended Sermons
+      </h2>
+      <ContentCarousel 
+        ref="sermonCar"
```
Comment (tt9) [inline src/home/pages/Home.vue]: Why are you using a ref for?

File: src/home/pages/Home.vue [hunk 3]
```diff
@@ -80,108 +124,242 @@
...
+  ];
+});
+
+watch(devoList, () => {
+  //@ts-expect-error
+  devoCar.value = devoList;
+})
+
+watch(sermonList, () => {
+  //@ts-expect-error
```
Comment (tt9) [inline src/home/pages/Home.vue]: What error is being thrown here, we shouldn't get in the habit of ignoring errors from typescript

File: src/devotionals/components/AuthorPreviewColumn.vue (hunk 1 above)
Comment (kenyonbowers) [reply src/devotionals/components/AuthorPreviewColumn.vue:29]: Ok

**Pull request merged**